### ISSUE TITLE
Substitution rule for `i.e.` and 'i. e.` does not work as expected.

### ISSUE THREAD

**Issue by ghost** [jdkato/txtlint#422]
## Issue

Vale does not detect `i.e.` or `i. e.` in the test file. 

## Environment

- OS: macOS Monterey 12.3
- Install method: Brew
- Vale version: 2.15.4
- Rule source: [LatinAbbreviations.yml](https://github.com/vultr/vultr-mdtk/blob/84e2213b281380f45c176b873cdf01cf24152623/styles/Vultr/LatinAbbreviations.yml)
- Test case: [Vultr.LatinAbbreviations.md](https://github.com/vultr/vultr-mdtk/blob/84e2213b281380f45c176b873cdf01cf24152623/tests/Vale/Vultr.LatinAbbreviations.md)

## Output
```
$ vale --output line Vultr.LatinAbbreviations.md

Vultr.LatinAbbreviations.md:12:35:Vultr.LatinAbbreviations:Please use 'for example' instead of 'e.g.', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:13:35:Vultr.LatinAbbreviations:Please use 'for example' instead of 'e. g.', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:15:51:Vultr.LatinAbbreviations:Please use 'and so on' instead of 'etc.', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:16:51:Vultr.LatinAbbreviations:Please use 'and so on' instead of 'et cetera', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:17:51:Vultr.LatinAbbreviations:Please use 'and so on' instead of 'etcetera', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:21:35:Vultr.LatinAbbreviations:Please use 'foobar testing' instead of 'z.z.', or consider rewriting the sentence.
Vultr.LatinAbbreviations.md:22:35:Vultr.LatinAbbreviations:Please use 'foobar testing' instead of 'z. z.', or consider rewriting the sentence.
```

**Comment by jdkato** [comment]
The problem is your use of brackets (`[...]`) in `Vultr/accept.txt`, such as:

https://github.com/vultr/vultr-mdtk/blob/84e2213b281380f45c176b873cdf01cf24152623/styles/Vocab/Vultr/accept.txt#L7

Parenthesis match against the enclosed words, while brackets match against the enclosed characters. So something like `[API|api]` isn't doing what you think; see https://regex101.com/r/ZxbTvN/1 for an example.
